gradle-scripts/docker.gradle

```diff
@@ -0,0 +1,13 @@
+docker {
+    javaApplication {
+        baseImage = 'java:openjdk-8-jre'
```

Comment:
why not `java:openjdk-8-jre-alpine` ?

Reply:
b73bde5 thanks!

Actually I used it without thinking. Didn't know that the alpine was that much smaller. Esp. that we (don't, right?) need the extra OS stuff in the normal one.